Makefile

```diff
@@ -659,8 +659,11 @@ dist/firecracker_kernel.tar.gz: firecracker/kernel/build.sh | dist
 # NOTE: While this target is PHONY, it *does* represent a real directory in
 # dist/
 .PHONY: dist/plugin-bootstrap-init
-dist/plugin-bootstrap-init: | dist  ## Build the Plugin Bootstrap Init (+ associated files) and copy it to dist/
-	$(DOCKER_BUILDX_BAKE_HCL) plugin-bootstrap-init
+dist/plugin-bootstrap-init: _shuttle-rust-to-dist  ## Build the Plugin Bootstrap Init (+ associated files) and copy it to dist/
+
+.PHONY: _shuttle-rust-to-dist
+_shuttle-rust-to-dist: | dist  ## Copy all specified Rust binary artifacts to dist/
```

Comment:
Can we call this something a little more descriptive, like maybe `extract-binary-artifacts`? I can't tell from the name what this does.

Reply:
Yeah, I'm cool with that. There are some artifacts that aren't strictly binaries tho 😱 but i'll play around with it